docker-compose.yml

```
@@ -0,0 +1,37 @@
+# Docker Compose file for the services needed by Ocim, for easy local development setup
+
+version: "3.6"
+
+services:
+
```

I also noticed that I needed to set up redis to be able to create new ovh instances. This requires you to set the REDIS_URL env var to "redis://redis:6379/" for this to reflect in my opencraft/settings.py
```suggestion
redis:
    image: redis:latest
    ports:
      - "6379:6379"
```

---

Yes, I threw the baby out with the bathwater on this one but just readded it and set it up as the default.
I force pushed though, so you might have to reset your local copy

---

We should probably also add redis to ocim's depends_on block